events/wc_dark_cult_events.txt

```diff
@@ -889,16 +888,33 @@ narrative_event = {
 	border = GFX_event_narrative_frame_intrigue
 
 	is_triggered_only = yes
-	
+
 	immediate = {
 		add_trait = travelling
 	}
 
 	# I will fulfill the will of the Gods!
 	option = {
 		name = EVTOPTA_WCDAC_5030
-
-		hidden_effect = { 73 = { province_event = { id = WCDAC.5034 days = 60 } } }
+		trigger_switch = {
+				on_trigger = has_character_modifier
+				follower_of_yogg_saron = { remove_character_modifier = follower_of_yogg_saron }
+				follower_of_nzoth = { remove_character_modifier = follower_of_nzoth }
+		}
+		add_character_modifier = { name = follower_of_cthun duration = -1 }
+		set_global_flag = twilights_hammer_kalimdor
+	}
+	option = {
+		name = EVTOPTB_WCDAC_5030
+		trigger_switch = {
+				on_trigger = has_character_modifier
+				follower_of_cthun = { remove_character_modifier = follower_of_cthun }
+				follower_of_nzoth = { remove_character_modifier = follower_of_nzoth }
+		}
+		add_character_modifier = { name = follower_of_yogg_saron duration = -1 }
+	}
+	after = {
+		hidden_effect = { 73 = { province_event = { id = WCDAC.5034 years = 16 } } }
```

Comment:
Break the timeline. It's said to take 4 years. Though, I wouldn't do it and assume Cho'gall made it in a few months as Jaina and Thrall did for gameplay reasons.
https://wow.gamepedia.com/Timeline_(unofficial)

Reply:
It took 4 to get to tanaris not silithus

Reply:
Give him province in Tanaris. As others said, gameplay-wise, it's boring to wait. You got an event to sail... aaand wait 4 years to get there. It's not what I expect from it because it doesn't look real at all.

Reply:
Well if you consider his crew its a miracle they got there at all

Reply:
I can add events about him landing in tanaris and garona and flavor about pale orc sailing

Reply:
If you mean the pale orcs, they got killed during the First War.
https://wow.gamepedia.com/Pale_orc#First_War

Reply:
It doesnt actually say anything about them dying

Reply:
They got sacrificed in ritual that freed cthun most likely

Reply:
> It doesnt actually say anything about them dying

> he branded the Twilight's Hammer's prophecies onto the skin of the Pale orcs

It says the Pale orcs... as a whole, soo... yep, they died.

Reply:
You realize pale orcs are twilights hammer clan? Like the whole clan is made up of them only